src/firebase-resource-manager.ts

```diff
@@ -0,0 +1,208 @@
+import * as clc from "cli-color";
```

Comment:
Since this file is pretty long in both of your PRs right now (`projects:create` and this one), how about splitting this file into `projects-create.ts` and `apps-create.ts`?

Oh, actually, it looks like the naming convention for the `src/` dir would make them `projectsCreate.ts` and `appsCreate.ts`.

Reply:
Yeah, I like this, avoiding merge conflict as the same time

Reply:
Done